src/java.xml/share/classes/module-info.java

```diff
@@ -416,6 +423,30 @@
  * </li>
  * </ul>
  *
+ * <h2 id="IN_CF">JAXP Configuration Files</h2>
+ * The JDK provides two <a href="#Conf_CF">JAXP Configuration Files</a>:
+ *
+ * <ul>
+ * <li id="IN_CF_DEFAULT"><a href="#Conf_CF_Default">{@code jaxp.properties}</a>:
+ * the default configuration that the JDK uses to set property values when XML
+ * factories are initiated.</li>
+ *
+ * <li id="IN_CF_STRICT">{@code jaxp-strict.properties}: a configuration that
+ * contains more restrictive settings than the default {@code jaxp.properties}.
+ * In particular:
+ *     <ul>
+ *     <li><a href="#JDKCATALOG_RESOLVE">JDKCatalog Resolve</a> is on "strict" setting </li>
+ *     <li><a href="#ExtFunc">Extension Functions</a> are disabled </li>
+ *     <li><a href="#EELimit">JAXP Limits</a> are set to smaller numbers </li>
+ *     </ul>
+ *
+ * Deploying with this configuration prevents processors from unknowingly making
+ * outbound network connections to fetch DTDs, or process XML that makes use of
```

Comment:
s/process/processing/

Reply:
In XML parlance, a "processor" is an aggregation of parsers, serializers, and other things that contribute to the processing. So I think it could be either here, but you have a point and if it stays as "processor" then it should link #FacPro where the term is defined.

Reply:
It's the wording that doesn't sound right, before this you have "making" which doesn't sound right with "process". Maybe it needs two sentences.

Reply:
Thanks! Same as Alan, I thought you were talking about "processor" as well on the first glance, then realized you're referring to the parallel statements :-)